django_guid/config.py

```diff
@@ -58,7 +59,12 @@ def integration_settings(self) -> IntegrationSettings:
 
     @property
     def uuid_length(self) -> int:
-        return self.settings.get('UUID_LENGTH', 32)
+        default_lenth: Dict[str, int] = defaultdict(lambda: 32, hex=32, string=36)
+        return self.settings.get('UUID_LENGTH', default_lenth[self.uuid_format])
```

Comment:
Would this be enough?
```suggestion
        default_length: Dict[str, int] = defaultdict(lambda: 32, string=36)
        return self.settings.get('UUID_LENGTH', default_length[self.uuid_format])
```

Reply:
Totally. Thank you.